parser/parse.go

```diff
@@ -392,9 +392,9 @@ func (b *Builder) importPackage(dir string, userRequested bool) (*tc.Package, er
 	if err != nil {
 		switch {
 		case ignoreError && pkg != nil:
-			klog.V(2).Infof("type checking encountered some issues in %q, but ignoring.\n", pkgPath)
+			klog.V(4).Infof("type checking encountered some issues in %q, but ignoring.\n", pkgPath)
 		case !ignoreError && pkg != nil:
-			klog.V(2).Infof("type checking encountered some errors in %q\n", pkgPath)
+			klog.V(3).Infof("type checking encountered some errors in %q\n", pkgPath)
```

Comment:
what are examples of these in practice? Type errors sound relevant for the user.

Reply:
Most of the type check errors are from built-in go libraries like `runtime`, `net/http`, etc. Most of these non-relevant errors are generated from packages that are [we are solely importing because they are referenced by other packages](https://github.com/kubernetes/gengo/blob/de9496dff47bef4ff8f45a56096bf50798cba2a5/parser/parse.go#L362-L364). We also eventually end up ignoring the type check errors generated from such packages but end up logging them today.

I have updated the PR to log such non-relevant errors at v=3 and log type check errors for packages we care about at v=2.

Example from k/k:

1. Logs before this change is applied at v=2 (not really readable):

<details><summary>Logs</summary>
<p>

```console
Generating openapi code for KUBE
I0630 18:42:18.622647 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622695 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:18.622711 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622719 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:18.622732 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622742 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:18.622758 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622770 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:18.622787 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622798 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:18.622887 4153647 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:18.627637 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.627676 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.627711 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:18.627837 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627850 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:18.627867 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627879 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:18.627894 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627904 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:18.627919 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627929 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:18.627944 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627954 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:18.627970 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627981 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:18.630589 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.630620 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:18.630798 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.630814 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:18.631496 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.631518 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:18.631818 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.631849 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:18.632156 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632181 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:18.632216 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632255 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632270 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:18.632730 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632749 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:18.632965 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632980 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:18.633141 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.633157 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:18.634028 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.634046 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:18.635269 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.635310 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:18.638087 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.638171 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:18.640128 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.640180 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:18.641295 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.641324 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:18.641342 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.641356 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:18.641384 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.641397 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:18.644870 4153647 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:18.737134 4153647 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:18.737331 4153647 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.737358 4153647 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:18.737583 4153647 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:19.009852 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:19.013596 4153647 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:19.304771 4153647 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:19.309757 4153647 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:19.313053 4153647 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:19.320610 4153647 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:19.328446 4153647 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:19.586895 4153647 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:19.596631 4153647 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:19.900268 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900366 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900500 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900615 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900681 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900751 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.905755 4153647 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:19.935722 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go:49:30: cannot use &(Unstructured literal) (value of type *Unstructured) as "k8s.io/apimachinery/pkg/runtime".Unstructured value in variable declaration: missing method DeepCopyObject
I0630 18:42:19.935959 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured_list.go:27:30: cannot use &(UnstructuredList literal) (value of type *UnstructuredList) as "k8s.io/apimachinery/pkg/runtime".Unstructured value in variable declaration: missing method DeepCopyObject
I0630 18:42:19.937398 4153647 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured", but ignoring.
I0630 18:42:21.540525 4153647 openapi.go:466] [k8s.io/api/apps/v1.DeploymentSpec] Strategy k8s.io/api/apps/v1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:21.615693 4153647 openapi.go:466] [k8s.io/api/apps/v1beta1.DeploymentSpec] Strategy k8s.io/api/apps/v1beta1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:21.704320 4153647 openapi.go:466] [k8s.io/api/apps/v1beta2.DeploymentSpec] Strategy k8s.io/api/apps/v1beta2.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:21.957718 4153647 openapi.go:466] [k8s.io/api/core/v1.PodSpec] Volumes []k8s.io/api/core/v1.Volume: [merge,retainKeys] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:22.066138 4153647 openapi.go:466] [k8s.io/api/extensions/v1beta1.DeploymentSpec] Strategy k8s.io/api/extensions/v1beta1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:22.176545 4153647 openapi.go:466] [k8s.io/api/policy/v1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: [replace] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:22.176560 4153647 openapi.go:466] [k8s.io/api/policy/v1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: tag patchStrategy on type Pointer; only allowed on type Slice
I0630 18:42:22.184961 4153647 openapi.go:466] [k8s.io/api/policy/v1beta1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: [replace] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:22.184975 4153647 openapi.go:466] [k8s.io/api/policy/v1beta1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: tag patchStrategy on type Pointer; only allowed on type Slice
I0630 18:42:22.343777 4153647 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:22.343797 4153647 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:24.937786 4153647 api_linter.go:44] Assembling file "api/api-rules/violation_exceptions.list"
Generating openapi code for AGGREGATOR
I0630 18:42:25.353272 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353320 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:25.353341 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353355 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:25.353373 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353387 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:25.353402 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353412 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:25.353429 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353440 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:25.353574 4153759 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:25.358878 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.358906 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.358941 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:25.359071 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359090 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:25.359103 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359109 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:25.359119 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359140 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:25.359157 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359167 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:25.359177 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359187 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:25.359202 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359211 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:25.361583 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.361607 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:25.361779 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.361794 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:25.362455 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.362473 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:25.362717 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.362734 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:25.362960 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.362977 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:25.362996 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.363021 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.363032 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:25.363661 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.363687 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:25.363986 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.364002 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:25.364173 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.364189 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:25.365247 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.365268 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:25.367309 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.367364 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:25.369274 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.369323 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:25.370679 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.370716 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:25.371714 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.371745 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:25.371768 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.371793 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:25.371827 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.371843 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:25.375899 4153759 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:25.456527 4153759 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:25.456702 4153759 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.456729 4153759 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:25.456977 4153759 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:25.705020 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.707824 4153759 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:25.854176 4153759 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:25.860066 4153759 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:26.147742 4153759 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:26.152508 4153759 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:26.155807 4153759 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:26.362976 4153759 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:26.373069 4153759 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:26.629957 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630001 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630026 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630049 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630071 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630096 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.631622 4153759 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:26.705597 4153759 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:26.705615 4153759 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:26.871264 4153759 api_linter.go:44] Assembling file "api/api-rules/aggregator_violation_exceptions.list"
Generating openapi code for APIEXTENSIONS
I0630 18:42:27.209661 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209726 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:27.209753 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209769 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:27.209786 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209801 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:27.209820 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209835 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:27.209852 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209863 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:27.210001 4153845 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:27.216310 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.216338 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.216383 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:27.216527 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216538 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:27.216556 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216565 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:27.216579 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216587 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:27.216600 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216608 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:27.216621 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216629 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:27.216641 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216659 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:27.219094 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.219124 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:27.219390 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.219410 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:27.220293 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220317 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:27.220598 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220622 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:27.220895 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220916 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:27.220955 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220982 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.221000 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:27.221469 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.221487 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:27.221788 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.221816 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:27.222104 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.222132 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:27.223695 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.223730 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:27.225402 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.225430 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:27.227182 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.227208 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:27.228433 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.228481 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:27.229491 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.229522 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:27.229539 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.229557 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:27.229591 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.229617 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:27.237329 4153845 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:27.327347 4153845 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:27.327492 4153845 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.327515 4153845 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:27.327740 4153845 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:27.555601 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.561972 4153845 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:27.943678 4153845 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:27.948840 4153845 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:28.207358 4153845 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:28.213180 4153845 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:28.217647 4153845 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:28.810139 4153845 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:28.821548 4153845 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:29.005178 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005245 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005274 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005298 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005321 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005343 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.007651 4153845 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:29.172380 4153845 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:29.172400 4153845 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:29.204485 4153845 api_linter.go:44] Assembling file "api/api-rules/apiextensions_violation_exceptions.list"
Generating openapi code for CODEGEN
I0630 18:42:29.694836 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694880 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:29.694894 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694902 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:29.694912 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694919 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:29.694930 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694940 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:29.694953 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694962 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:29.695040 4153932 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:29.700211 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.700244 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.700271 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:29.700415 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700428 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:29.700446 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700455 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:29.700470 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700479 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:29.700494 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700502 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:29.700519 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700528 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:29.700542 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700551 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:29.702862 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.702884 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:29.703050 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.703063 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:29.703729 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.703747 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:29.704006 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704024 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:29.704260 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704283 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:29.704307 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704332 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704351 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:29.704818 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704839 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:29.705064 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.705079 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:29.705227 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.705256 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:29.706167 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.706184 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:29.708619 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.708675 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:29.710771 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.710826 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:29.711989 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712015 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:29.712849 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712874 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:29.712896 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712923 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:29.712951 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712966 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:29.717048 4153932 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:29.798314 4153932 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:29.798494 4153932 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.798525 4153932 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:29.798795 4153932 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:30.079744 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:30.083602 4153932 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:30.230199 4153932 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:30.236008 4153932 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:30.526495 4153932 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:30.531748 4153932 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:30.535804 4153932 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:30.774565 4153932 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:30.785621 4153932 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:31.084233 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084273 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084290 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084312 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084339 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084357 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.085902 4153932 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:31.144136 4153932 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:31.144152 4153932 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:31.292874 4153932 api_linter.go:44] Assembling file "api/api-rules/codegen_violation_exceptions.list"
Generating openapi code for SAMPLEAPISERVER
I0630 18:42:31.697023 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697069 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:31.697090 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697105 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:31.697121 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697129 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:31.697141 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697153 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:31.697163 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697170 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:31.697289 4154042 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:31.702200 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.702255 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.702301 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:31.702439 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702451 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:31.702464 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702476 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:31.702493 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702509 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:31.702531 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702543 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:31.702562 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702574 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:31.702596 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702608 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:31.705040 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.705066 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:31.705273 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.705287 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:31.706105 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706130 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:31.706555 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706578 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:31.706837 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706858 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:31.706888 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706921 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706936 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:31.707410 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.707427 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:31.707717 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.707734 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:31.707943 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.707961 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:31.709291 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.709313 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:31.713005 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.713076 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:31.718516 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.718642 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:31.721641 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.721687 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:31.724316 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.724352 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:31.724391 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.724426 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:31.724469 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.724491 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:31.736851 4154042 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:31.819437 4154042 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:31.819690 4154042 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.819729 4154042 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:31.820039 4154042 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:32.048739 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:32.051115 4154042 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:32.248127 4154042 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:32.253432 4154042 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:32.548069 4154042 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:32.562469 4154042 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:32.567904 4154042 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:32.778787 4154042 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:32.788955 4154042 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:32.948861 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948899 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948925 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948948 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948980 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.949003 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:33.138301 4154042 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:33.332547 4154042 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:33.332570 4154042 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:33.484497 4154042 api_linter.go:44] Assembling file "api/api-rules/sample_apiserver_violation_exceptions.list"

```

</p>
</details>

2. Logs after this change is applied at v=2 (much more readable):

<details><summary>Logs</summary>
<p>

```console
Generating openapi code for KUBE
I0630 18:42:47.072086 4155342 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:47.097174 4155342 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:47.186415 4155342 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:47.467644 4155342 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:47.755346 4155342 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:47.769791 4155342 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:47.979326 4155342 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:48.409213 4155342 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:48.414639 4155342 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured", but ignoring.
I0630 18:42:50.000020 4155342 openapi.go:466] [k8s.io/api/apps/v1.DeploymentSpec] Strategy k8s.io/api/apps/v1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:50.035944 4155342 openapi.go:466] [k8s.io/api/apps/v1beta1.DeploymentSpec] Strategy k8s.io/api/apps/v1beta1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:50.065846 4155342 openapi.go:466] [k8s.io/api/apps/v1beta2.DeploymentSpec] Strategy k8s.io/api/apps/v1beta2.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:50.335952 4155342 openapi.go:466] [k8s.io/api/core/v1.PodSpec] Volumes []k8s.io/api/core/v1.Volume: [merge,retainKeys] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:50.463593 4155342 openapi.go:466] [k8s.io/api/extensions/v1beta1.DeploymentSpec] Strategy k8s.io/api/extensions/v1beta1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:50.596269 4155342 openapi.go:466] [k8s.io/api/policy/v1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: [replace] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:50.596286 4155342 openapi.go:466] [k8s.io/api/policy/v1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: tag patchStrategy on type Pointer; only allowed on type Slice
I0630 18:42:50.621934 4155342 openapi.go:466] [k8s.io/api/policy/v1beta1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: [replace] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:50.621956 4155342 openapi.go:466] [k8s.io/api/policy/v1beta1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: tag patchStrategy on type Pointer; only allowed on type Slice
I0630 18:42:50.989672 4155342 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:50.989691 4155342 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:53.492458 4155342 api_linter.go:44] Assembling file "api/api-rules/violation_exceptions.list"
Generating openapi code for AGGREGATOR
I0630 18:42:53.866496 4155451 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:53.909295 4155451 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:53.985401 4155451 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:54.269811 4155451 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:54.425284 4155451 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:54.685695 4155451 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:54.896161 4155451 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:55.156617 4155451 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:55.284456 4155451 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:55.284473 4155451 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:55.435734 4155451 api_linter.go:44] Assembling file "api/api-rules/aggregator_violation_exceptions.list"
Generating openapi code for APIEXTENSIONS
I0630 18:42:55.776678 4155537 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:55.813287 4155537 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:55.898980 4155537 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:56.154563 4155537 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:56.532884 4155537 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:56.811768 4155537 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:57.037012 4155537 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:57.217353 4155537 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:57.451026 4155537 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:57.451090 4155537 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:57.637070 4155537 api_linter.go:44] Assembling file "api/api-rules/apiextensions_violation_exceptions.list"
Generating openapi code for CODEGEN
I0630 18:42:57.990917 4155623 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:58.037100 4155623 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:58.119485 4155623 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:58.349063 4155623 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:58.531785 4155623 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:58.799521 4155623 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:59.043131 4155623 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:59.219991 4155623 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:59.411248 4155623 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:59.411330 4155623 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:59.570275 4155623 api_linter.go:44] Assembling file "api/api-rules/codegen_violation_exceptions.list"
Generating openapi code for SAMPLEAPISERVER
I0630 18:42:59.949407 4155709 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:59.995047 4155709 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:43:00.074607 4155709 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:43:00.306718 4155709 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:43:00.484730 4155709 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:43:00.769918 4155709 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:43:00.993101 4155709 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:43:01.274344 4155709 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:43:01.453767 4155709 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:43:01.453786 4155709 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:43:01.594737 4155709 api_linter.go:44] Assembling file "api/api-rules/sample_apiserver_violation_exceptions.list"

```

</p>
</details>

3. Logs after this change is applied at v=3 (includes type check errors from non-relevant packages as well):

<details><summary>Logs</summary>
<p>

```console
Go version: go version go1.16.4 linux/amd64
+++ [0630 18:41:57] Building go targets for linux/amd64:
    ./vendor/k8s.io/kube-openapi/cmd/openapi-gen
Env for linux/amd64: GOOS=linux GOARCH=amd64 GOROOT=/usr/local/go CGO_ENABLED= CC=
Coverage is disabled.
+++ [0630 18:42:18] Placing binaries
Generating openapi code for KUBE
I0630 18:42:18.622647 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622695 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:18.622711 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622719 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:18.622732 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622742 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:18.622758 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622770 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:18.622787 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.622798 4153647 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:18.622887 4153647 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:18.627637 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.627676 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.627711 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:18.627837 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627850 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:18.627867 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627879 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:18.627894 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627904 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:18.627919 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627929 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:18.627944 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627954 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:18.627970 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.627981 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:18.630589 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.630620 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:18.630798 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:18.630814 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:18.631496 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.631518 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:18.631818 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.631849 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:18.632156 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632181 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:18.632216 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632255 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632270 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:18.632730 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632749 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:18.632965 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.632980 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:18.633141 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.633157 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:18.634028 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.634046 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:18.635269 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.635310 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:18.638087 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.638171 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:18.640128 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.640180 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:18.641295 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.641324 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:18.641342 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.641356 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:18.641384 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.641397 4153647 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:18.644870 4153647 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:18.737134 4153647 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:18.737331 4153647 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:18.737358 4153647 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:18.737583 4153647 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:19.009852 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:19.013596 4153647 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:19.304771 4153647 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:19.309757 4153647 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:19.313053 4153647 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:19.320610 4153647 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:19.328446 4153647 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:19.586895 4153647 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:19.596631 4153647 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:19.900268 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900366 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900500 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900615 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900681 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.900751 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:19.905755 4153647 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:19.935722 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go:49:30: cannot use &(Unstructured literal) (value of type *Unstructured) as "k8s.io/apimachinery/pkg/runtime".Unstructured value in variable declaration: missing method DeepCopyObject
I0630 18:42:19.935959 4153647 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured_list.go:27:30: cannot use &(UnstructuredList literal) (value of type *UnstructuredList) as "k8s.io/apimachinery/pkg/runtime".Unstructured value in variable declaration: missing method DeepCopyObject
I0630 18:42:19.937398 4153647 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured", but ignoring.
I0630 18:42:21.540525 4153647 openapi.go:466] [k8s.io/api/apps/v1.DeploymentSpec] Strategy k8s.io/api/apps/v1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:21.615693 4153647 openapi.go:466] [k8s.io/api/apps/v1beta1.DeploymentSpec] Strategy k8s.io/api/apps/v1beta1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:21.704320 4153647 openapi.go:466] [k8s.io/api/apps/v1beta2.DeploymentSpec] Strategy k8s.io/api/apps/v1beta2.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:21.957718 4153647 openapi.go:466] [k8s.io/api/core/v1.PodSpec] Volumes []k8s.io/api/core/v1.Volume: [merge,retainKeys] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:22.066138 4153647 openapi.go:466] [k8s.io/api/extensions/v1beta1.DeploymentSpec] Strategy k8s.io/api/extensions/v1beta1.DeploymentStrategy: tag patchStrategy on type Struct; only allowed on type Slice
I0630 18:42:22.176545 4153647 openapi.go:466] [k8s.io/api/policy/v1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: [replace] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:22.176560 4153647 openapi.go:466] [k8s.io/api/policy/v1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: tag patchStrategy on type Pointer; only allowed on type Slice
I0630 18:42:22.184961 4153647 openapi.go:466] [k8s.io/api/policy/v1beta1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: [replace] not allowed for patchStrategy. Allowed values: [merge retainKeys]
I0630 18:42:22.184975 4153647 openapi.go:466] [k8s.io/api/policy/v1beta1.PodDisruptionBudgetSpec] Selector *k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelector: tag patchStrategy on type Pointer; only allowed on type Slice
I0630 18:42:22.343777 4153647 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:22.343797 4153647 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:24.937786 4153647 api_linter.go:44] Assembling file "api/api-rules/violation_exceptions.list"
Generating openapi code for AGGREGATOR
I0630 18:42:25.353272 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353320 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:25.353341 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353355 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:25.353373 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353387 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:25.353402 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353412 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:25.353429 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.353440 4153759 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:25.353574 4153759 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:25.358878 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.358906 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.358941 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:25.359071 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359090 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:25.359103 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359109 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:25.359119 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359140 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:25.359157 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359167 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:25.359177 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359187 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:25.359202 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.359211 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:25.361583 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.361607 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:25.361779 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:25.361794 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:25.362455 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.362473 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:25.362717 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.362734 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:25.362960 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.362977 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:25.362996 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.363021 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.363032 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:25.363661 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.363687 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:25.363986 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.364002 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:25.364173 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.364189 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:25.365247 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.365268 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:25.367309 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.367364 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:25.369274 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.369323 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:25.370679 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.370716 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:25.371714 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.371745 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:25.371768 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.371793 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:25.371827 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.371843 4153759 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:25.375899 4153759 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:25.456527 4153759 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:25.456702 4153759 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.456729 4153759 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:25.456977 4153759 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:25.705020 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:25.707824 4153759 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:25.854176 4153759 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:25.860066 4153759 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:26.147742 4153759 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:26.152508 4153759 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:26.155807 4153759 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:26.362976 4153759 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:26.373069 4153759 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:26.629957 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630001 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630026 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630049 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630071 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.630096 4153759 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:26.631622 4153759 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:26.705597 4153759 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:26.705615 4153759 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:26.871264 4153759 api_linter.go:44] Assembling file "api/api-rules/aggregator_violation_exceptions.list"
Generating openapi code for APIEXTENSIONS
I0630 18:42:27.209661 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209726 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:27.209753 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209769 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:27.209786 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209801 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:27.209820 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209835 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:27.209852 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.209863 4153845 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:27.210001 4153845 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:27.216310 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.216338 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.216383 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:27.216527 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216538 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:27.216556 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216565 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:27.216579 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216587 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:27.216600 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216608 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:27.216621 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216629 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:27.216641 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.216659 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:27.219094 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.219124 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:27.219390 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:27.219410 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:27.220293 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220317 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:27.220598 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220622 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:27.220895 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220916 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:27.220955 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.220982 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.221000 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:27.221469 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.221487 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:27.221788 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.221816 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:27.222104 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.222132 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:27.223695 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.223730 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:27.225402 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.225430 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:27.227182 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.227208 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:27.228433 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.228481 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:27.229491 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.229522 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:27.229539 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.229557 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:27.229591 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.229617 4153845 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:27.237329 4153845 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:27.327347 4153845 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:27.327492 4153845 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.327515 4153845 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:27.327740 4153845 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:27.555601 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:27.561972 4153845 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:27.943678 4153845 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:27.948840 4153845 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:28.207358 4153845 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:28.213180 4153845 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:28.217647 4153845 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:28.810139 4153845 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:28.821548 4153845 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:29.005178 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005245 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005274 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005298 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005321 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.005343 4153845 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:29.007651 4153845 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:29.172380 4153845 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:29.172400 4153845 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:29.204485 4153845 api_linter.go:44] Assembling file "api/api-rules/apiextensions_violation_exceptions.list"
Generating openapi code for CODEGEN
I0630 18:42:29.694836 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694880 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:29.694894 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694902 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:29.694912 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694919 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:29.694930 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694940 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:29.694953 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.694962 4153932 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:29.695040 4153932 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:29.700211 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.700244 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.700271 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:29.700415 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700428 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:29.700446 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700455 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:29.700470 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700479 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:29.700494 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700502 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:29.700519 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700528 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:29.700542 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.700551 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:29.702862 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.702884 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:29.703050 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:29.703063 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:29.703729 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.703747 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:29.704006 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704024 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:29.704260 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704283 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:29.704307 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704332 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704351 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:29.704818 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.704839 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:29.705064 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.705079 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:29.705227 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.705256 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:29.706167 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.706184 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:29.708619 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.708675 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:29.710771 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.710826 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:29.711989 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712015 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:29.712849 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712874 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:29.712896 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712923 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:29.712951 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.712966 4153932 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:29.717048 4153932 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:29.798314 4153932 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:29.798494 4153932 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:29.798525 4153932 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:29.798795 4153932 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:30.079744 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:30.083602 4153932 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:30.230199 4153932 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:30.236008 4153932 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:30.526495 4153932 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:30.531748 4153932 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:30.535804 4153932 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:30.774565 4153932 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:30.785621 4153932 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:31.084233 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084273 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084290 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084312 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084339 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.084357 4153932 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:31.085902 4153932 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:31.144136 4153932 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:31.144152 4153932 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:31.292874 4153932 api_linter.go:44] Assembling file "api/api-rules/codegen_violation_exceptions.list"
Generating openapi code for SAMPLEAPISERVER
I0630 18:42:31.697023 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:39: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697069 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:14:23: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:31.697090 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:39: cannot use cpu.X86.HasSSE42 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697105 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:15:23: unsafe.Offsetof(cpu.X86.HasSSE42) (value of type uintptr) is not constant
I0630 18:42:31.697121 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:39: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697129 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:16:23: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:31.697141 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:39: cannot use cpu.X86.HasPOPCNT (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697153 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:17:23: unsafe.Offsetof(cpu.X86.HasPOPCNT) (value of type uintptr) is not constant
I0630 18:42:31.697163 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:37: cannot use cpu.S390X.HasVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.697170 4154042 parse.go:449] type checker: /usr/local/go/src/internal/bytealg/bytealg.go:19:21: unsafe.Offsetof(cpu.S390X.HasVX) (value of type uintptr) is not constant
I0630 18:42:31.697289 4154042 parse.go:395] type checking encountered some issues in "internal/bytealg", but ignoring.
I0630 18:42:31.702200 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.702255 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:66: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.702301 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/chan.go:28:14: unsafe.Sizeof((hchan literal)) + uintptr(-int(unsafe.Sizeof((hchan literal))) & (maxAlign - 1)) (value of type uintptr) is not constant
I0630 18:42:31.702439 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:37: cannot use cpu.X86.HasAVX (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702451 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:14:21: unsafe.Offsetof(cpu.X86.HasAVX) (value of type uintptr) is not constant
I0630 18:42:31.702464 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:37: cannot use cpu.X86.HasAVX2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702476 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:15:21: unsafe.Offsetof(cpu.X86.HasAVX2) (value of type uintptr) is not constant
I0630 18:42:31.702493 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:37: cannot use cpu.X86.HasERMS (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702509 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:16:21: unsafe.Offsetof(cpu.X86.HasERMS) (value of type uintptr) is not constant
I0630 18:42:31.702531 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:37: cannot use cpu.X86.HasSSE2 (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702543 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:17:21: unsafe.Offsetof(cpu.X86.HasSSE2) (value of type uintptr) is not constant
I0630 18:42:31.702562 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:38: cannot use cpu.ARM.HasIDIVA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702574 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:19:22: unsafe.Offsetof(cpu.ARM.HasIDIVA) (value of type uintptr) is not constant
I0630 18:42:31.702596 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:40: cannot use cpu.MIPS64X.HasMSA (variable of type bool) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.702608 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/cpuflags.go:21:24: unsafe.Offsetof(cpu.MIPS64X.HasMSA) (value of type uintptr) is not constant
I0630 18:42:31.705040 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:50: cannot use (mcentral literal) (value of type mcentral) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.705066 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:208:13: array length cpu.CacheLinePadSize - unsafe.Sizeof((mcentral literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:31.705273 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:31: cannot use (struct{b bmap; v int64} literal).v (value of type int64) as unsafe.ArbitraryType value in argument to unsafe.Offsetof
I0630 18:42:31.705287 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/map.go:83:15: unsafe.Offsetof((struct{b bmap; v int64} literal).v) (value of type uintptr) is not constant
I0630 18:42:31.706105 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:65: cannot use (finalizer literal) (value of type finalizer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706130 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mfinal.go:28:11: array length (_FinBlockSize - 2 * sys.PtrSize - 2 * 4) / unsafe.Sizeof((finalizer literal)) (value of type uintptr) must be constant
I0630 18:42:31.706555 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706578 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcwork.go:325:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:31.706837 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:37: cannot use (stackWorkBufHdr literal) (value of type stackWorkBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706858 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:110:7: array length (_WorkbufSize - unsafe.Sizeof((stackWorkBufHdr literal))) / sys.PtrSize (value of type uintptr) must be constant
I0630 18:42:31.706888 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:37: cannot use (stackObjectBufHdr literal) (value of type stackObjectBufHdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706921 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:75: cannot use (stackObject literal) (value of type stackObject) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.706936 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mgcstack.go:130:7: array length (_WorkbufSize - unsafe.Sizeof((stackObjectBufHdr literal))) / unsafe.Sizeof((stackObject literal)) (value of type uintptr) must be constant
I0630 18:42:31.707410 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:41: cannot use (gcBitsHeader literal) (value of type gcBitsHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.707427 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mheap.go:1887:27: unsafe.Sizeof((gcBitsHeader literal)) (value of type uintptr) is not constant
I0630 18:42:31.707717 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:33: cannot use pallocSum(0) (constant 0 of type pallocSum) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.707734 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagealloc.go:902:19: unsafe.Sizeof(pallocSum(0)) (value of type uintptr) is not constant
I0630 18:42:31.707943 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:42: cannot use (pageCache literal).cache (value of type uint64) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.707961 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/mpagecache.go:12:24: 8 * unsafe.Sizeof((pageCache literal).cache) (value of type uintptr) is not constant
I0630 18:42:31.709291 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:34: cannot use (_defer literal) (value of type _defer) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.709313 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/panic.go:318:20: unsafe.Sizeof((_defer literal)) (value of type uintptr) is not constant
I0630 18:42:31.713005 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:45: cannot use (semaRoot literal) (value of type semaRoot) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.713076 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/sema.go:51:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((semaRoot literal)) (value of type uintptr) must be constant
I0630 18:42:31.718516 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:45: cannot use (stackpoolItem literal) (value of type stackpoolItem) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.718642 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/stack.go:143:8: array length cpu.CacheLinePadSize - unsafe.Sizeof((stackpoolItem literal)) % cpu.CacheLinePadSize (value of type uintptr) must be constant
I0630 18:42:31.721641 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.721687 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/trace.go:160:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
I0630 18:42:31.724316 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:52: cannot use (elfSym literal) (value of type elfSym) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.724352 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:54:23: vdsoArrayMax / unsafe.Sizeof((elfSym literal)) (value of type uintptr) is not constant
I0630 18:42:31.724391 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:52: cannot use (elfDyn literal) (value of type elfDyn) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.724426 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:55:23: vdsoArrayMax / unsafe.Sizeof((elfDyn literal)) (value of type uintptr) is not constant
I0630 18:42:31.724469 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:37: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.724491 4154042 parse.go:449] type checker: /usr/local/go/src/runtime/vdso_linux.go:62:23: unsafe.Sizeof(uintptr(0)) / 4 (value of type uintptr) is not constant
I0630 18:42:31.736851 4154042 parse.go:395] type checking encountered some issues in "runtime", but ignoring.
I0630 18:42:31.819437 4154042 parse.go:449] type checker: /usr/local/go/src/sync/map.go:70:31: cannot convert new(interface{}) (value of type *interface{}) to unsafe.Pointer
I0630 18:42:31.819690 4154042 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:27: cannot use (poolLocalInternal literal) (value of type poolLocalInternal) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:31.819729 4154042 parse.go:449] type checker: /usr/local/go/src/sync/pool.go:70:7: array length 128 - unsafe.Sizeof((poolLocalInternal literal)) % 128 (value of type uintptr) must be constant
I0630 18:42:31.820039 4154042 parse.go:395] type checking encountered some issues in "sync", but ignoring.
I0630 18:42:32.048739 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/gogo/protobuf/proto/pointer_unsafe.go:75:29: cannot use uintptr(0) (constant 0 of type uintptr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
I0630 18:42:32.051115 4154042 parse.go:395] type checking encountered some issues in "github.com/gogo/protobuf/proto", but ignoring.
I0630 18:42:32.248127 4154042 parse.go:449] type checker: /usr/local/go/src/net/dnsclient.go:10:2: could not import golang.org/x/net/dns/dnsmessage (Config.Importer.Import(golang.org/x/net/dns/dnsmessage) returned nil but no error)
I0630 18:42:32.253432 4154042 parse.go:395] type checking encountered some issues in "net", but ignoring.
I0630 18:42:32.548069 4154042 parse.go:449] type checker: /usr/local/go/src/crypto/tls/cipher_suites.go:20:2: could not import golang.org/x/crypto/chacha20poly1305 (Config.Importer.Import(golang.org/x/crypto/chacha20poly1305) returned nil but no error)
I0630 18:42:32.562469 4154042 parse.go:449] type checker: /usr/local/go/src/crypto/tls/key_schedule.go:17:2: could not import golang.org/x/crypto/hkdf (Config.Importer.Import(golang.org/x/crypto/hkdf) returned nil but no error)
I0630 18:42:32.567904 4154042 parse.go:395] type checking encountered some issues in "crypto/tls", but ignoring.
I0630 18:42:32.778787 4154042 parse.go:449] type checker: /usr/local/go/src/net/http/transport.go:34:2: could not import golang.org/x/net/http/httpproxy (Config.Importer.Import(golang.org/x/net/http/httpproxy) returned nil but no error)
I0630 18:42:32.788955 4154042 parse.go:395] type checking encountered some issues in "net/http", but ignoring.
I0630 18:42:32.948861 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:57:2: cannot use &(ListOptions literal) (value of type *ListOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948899 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:58:2: cannot use &(GetOptions literal) (value of type *GetOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948925 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:59:2: cannot use &(DeleteOptions literal) (value of type *DeleteOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948948 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:60:2: cannot use &(CreateOptions literal) (value of type *CreateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.948980 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:61:2: cannot use &(UpdateOptions literal) (value of type *UpdateOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:32.949003 4154042 parse.go:449] type checker: /home/nikhita/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/register.go:62:2: cannot use &(PatchOptions literal) (value of type *PatchOptions) as "k8s.io/apimachinery/pkg/runtime".Object value in array or slice literal: missing method DeepCopyObject
I0630 18:42:33.138301 4154042 parse.go:395] type checking encountered some issues in "k8s.io/apimachinery/pkg/apis/meta/v1", but ignoring.
I0630 18:42:33.332547 4154042 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchMergeKey on type Builtin; only allowed on type Slice
I0630 18:42:33.332570 4154042 openapi.go:466] [k8s.io/apimachinery/pkg/apis/meta/v1.LabelSelectorRequirement] Key string: tag patchStrategy on type Builtin; only allowed on type Slice
I0630 18:42:33.484497 4154042 api_linter.go:44] Assembling file "api/api-rules/sample_apiserver_violation_exceptions.list"
```

</p>
</details>

Reply:
> what are examples of these in practice? 

Example from k/k - https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/persistentvolume/pv_controller.go#L1015-L1035

Reply:
ack